src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -2698,4 +2698,76 @@ limitations under the License.
             <include cluster="Device Energy Management Mode" client="false" server="true" clientLocked="true" serverLocked="false"></include>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>MA-heatpump</name>
+        <domain>CHIP</domain>
+        <typeName>Heat Pump</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x0309</deviceId>
+        <class>Simple</class>
+        <scope>Endpoint</scope>
+        <clusters>
```

Comment:
I'm still learning how to map the spec to the code but It looks like there are a few mandatory clusters missing, according to: https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/device_types/HeatPump.adoc#device-type-requirements

ElectricalSensor and DEM at least, I believe.

Reply:
Nevermind, I was looking at the wrong list. This one looks right: https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/device_types/HeatPump.adoc#cluster-requirements

Reply:
Correct - the device type stuff can be confusing - the Electrical Sensor and DEM are also device types (with their own cluster requirements) that need to be included if this device type is included.